cmd/tsbs_generate_queries/databases/redistimeseries/devops.go

```diff
@@ -0,0 +1,145 @@
+package redistimeseries
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/timescale/tsbs/cmd/tsbs_generate_queries/uses/devops"
+	"github.com/timescale/tsbs/query"
+)
+
+// TODO: Remove the need for this by continuing to bubble up errors
+func panicIfErr(err error) {
```

Comment:
From the official docs:

// The panic built-in function stops normal execution of the current
// goroutine. When a function F calls panic, normal execution of F stops
// immediately. Any functions whose execution was deferred by F are run in
// the usual way, and then F returns to its caller. To the caller G, the
// invocation of F then behaves like a call to panic, terminating G's
// execution and running any deferred functions.

I think we should use this to prevent function calls errors within goroutines to pass by without being noticed.